agave/tasks/sqs_client.py

```
@@ -0,0 +1,63 @@
+from dataclasses import dataclass, field
+from typing import Optional, Union
+from uuid import uuid4
+
+try:
+    from aiobotocore.session import get_session
+    from types_aiobotocore_sqs import SQSClient
+except ImportError:
+    raise ImportError(
+        "You must install agave with [fastapi, tasks] option.\n"
+        "You can install it with: pip install agave[fastapi, tasks]"
+    )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)